### ISSUE TITLE
Restart App does not close bug page

### ISSUE THREAD

**Issue by imtrobin** [Ereza/CustomActivityOnCrash#65]
Hi, when I click on Restart App, I noticed that page with the bug is still in the background after the new app is launch. Is that intended or should it auto exit when I restart?

**Comment by Ereza** [comment]
It should not happen. There's probably something wrong with your configuration. Maybe the `ErrorActivity` does not have a `process` specified? Or the restart activity is wrongly defined?

Please reopen if you can provide more info about this (a project demonstrating the issue would be great!). Thanks!

**Comment by imtrobin** [comment]
Ok, I updated to 2.30. Here's an apk demonstrating the problem, wait for 5 second, it will crash. This is a default cocos2dx project, I just modified onCreate to crash after 5 seconds. When I click Restart App, before it crash again, go look at the list of process and you will find the bug page is still in background.

Repro
1. Launch app
2. Wait 5 second , it will crash
3. Restart App
4. Before it crash again after 5 seconds, check list of background process.

https://filedn.com/l3TGy7Y83c247u0RDYa9fkp/temp/code/testcrash.apk

Source
https://filedn.com/l3TGy7Y83c247u0RDYa9fkp/temp/code/testcrash.7z
